src/jvm/main/org/jetbrains/kotlinx/lincheck/strategy/managed/ManagedStrategy.kt

```diff
@@ -864,7 +869,7 @@ private class MonitorTracker(nThreads: Int) {
     fun releaseMonitor(monitor: Any) {
         // Decrement the reentrancy depth and remove the acquisition info
         // if the monitor becomes free to acquire by another thread.
-        val ai = acquiredMonitors[monitor]!!
+        val ai = acquiredMonitors[monitor] ?: return
```

Comment:
How can you invoke `release` without `acquire`?

Reply:
I got an execution exception overrided by `NullPointerException` here, but this situation should not happen, so I reverted the change